code/modules/mob/living/carbon/human/species/station/monkey.dm

```diff
@@ -40,7 +40,7 @@
 	push_flags = MONKEY|SLIME|SIMPLE_ANIMAL|ALIEN
 
 	pass_flags = PASSTABLE
-	holder_type = /obj/item/weapon/holder
```

Comment:
holder types for human mobs are defined in the species now, so thats why they were broken